code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -0,0 +1,347 @@
+/obj/effect/proc_holder/spell
```

Comment:
just make a spell subtype seriously

Reply:
why is he redefining proc_holder/spell outside of its files.